packages/vanilla/src/components/global-nav/top-nav/top-nav.scss

```diff
@@ -21,22 +21,30 @@ $group-height: 56px;
   @include shadow("dropshadow");
 }
 
-.hig__global-nav__top-nav__item {
+.hig__global-nav__top-nav__items {
```

Comment:
There's some refactoring of the menu items to make the padding look natural regardless of whether the hamburger is visible.

Reply:
Looks like I need to revert this for now; tests for other nav items are breaking.